packages/react-jsx-highcharts/test/components/Series/SeriesTypes.integration.spec.js

```diff
@@ -92,6 +104,9 @@ Object.keys(all).filter(name => /^[A-Z].*Series$/.test(name)).forEach((seriesNam
         };
         const WithComponent = withHighcharts(Component, Highcharts);
         const renderedChart = renderIntoDocument(<WithComponent />);
+        act(() => {
+          jest.runAllTimers();
```

Comment:
Using fake timers slows down these tests 3-4 times = 30s. Hopefully this can be fixed somehow.